opensrp-fp/src/main/java/org/smartregister/fp/features/home/repository/PatientRepository.java

```diff
@@ -184,4 +183,20 @@ public static void updateNextContactDate(String nextContactDate, String baseEnti
                 new String[]{baseEntityId});
     }
 
+    public static void doArchive(String baseEntityId) {
+        ContentValues contentValues = new ContentValues();
+        contentValues.put(DBConstantsUtils.KeyUtils.ARCHIVED, "yes");
```

Comment:
Please use constants instead of hard coded strings

Reply:
Done